### PR TITLE
Fusion inventory: Init at 2.3.18

### PR DESCRIPTION
**Pull request by phile314**
###### Motivation for this change

Adds a package for the fusion inventory software and the corresponding NixOS service. Fusion Inventory is a monitoring agent.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/services/monitoring/fusion-inventory.nix [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+      enable = mkOption {
+        default = false;
+        description = ''
+          Whether to run the Fusion Inventory agent on this machine.
+        '';
+      };
+
+      servers = mkOption {
+        type = types.listOf types.string;
```
Comment (joachifm) [inline nixos/modules/services/monitoring/fusion-inventory.nix]: Please use `types.str`

File: nixos/modules/services/monitoring/fusion-inventory.nix [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+in {
+
+  ###### interface
+
+  options = {
+
+    services.fusionInventory = {
+
+      enable = mkOption {
+        default = false;
```
Comment (joachifm) [inline nixos/modules/services/monitoring/fusion-inventory.nix]: Please declare a type.

File: pkgs/servers/monitoring/fusion-inventory/default.nix [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{ stdenv, fetchurl, gnused, buildPerlPackage, perl, perlPackages
```
Comment (joachifm) [inline pkgs/servers/monitoring/fusion-inventory/default.nix]: The `gnused` and `perl` parameters appear to be unused.

Comment (joachifm) [reply pkgs/servers/monitoring/fusion-inventory/default.nix]: That's the last nitpick I have on this :)

**Comment by phile314** [comment]
Rebased it on current nixos-unstable. Can we merge this?

**Comment by phile314** [comment]
Thanks :+1:

**Pull request merged**